### DISCUSSION TITLE
New tv shows

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#316] [General]
Do you know some good new tv shows worth watching?

**Comment by hernessteve** [comment]
Little fires everywhere

**Comment by Ozankey** [comment]
How I met your father. Haven't seen yet, but curiouscause I like How I met your mother. Hope it'll be the same interesting.

**Comment by ellielo** [comment]
Thanks for the recommendations guys. I received a firestick as a Christmas gift this year and trying to set up everything by myself. Any tips? So far I found out that it is important to hide my ip address while streaming [https://www.firesticktricks.com/hide-ip-address.html](https://www.firesticktricks.com/hide-ip-address.html), something else I should be aware of?
